src_types/common/anyString$wSelector.d.ts

```diff
@@ -0,0 +1,4 @@
+declare namespace $w {
```

Comment:
1. i would move the file into the "public" folder (same as "pages" has a $w file) since it's not "common".
2. what is the `void$w$w.d.ts` in the "common" folder ? does it relate ?